README.md

```diff
@@ -9,109 +8,111 @@ open-power-host-os/builds repository is an open source collaboration
 effort that aims to help administrators to build and deploy the latest
 and greatest capabilities in the OpenPOWER world through a build
 script that provides software well packaged and designed for the
-`Supported GNU/Linux distributions`_
+`Supported GNU/Linux distributions`.
```

Comment:
The `` have no use here. According to http://stackoverflow.com/a/15893175, you can link a section with `[Supported GNU/Linux distributions](#supported-gnulinux-distributions)`.